apps/osim/tests/test_models.py

```diff
@@ -668,37 +689,26 @@ def test_adjust(self):
                 "states": [],  # this is not valid but OK for this test
             }
         )
-        workflow.states = random_states
+        workflow.states = states
         workflow_framework.register_workflow(workflow)
 
-        flaw = FlawFactory.build(major_incident_state=Flaw.FlawMajorIncident.APPROVED)
-        flaw.save(raise_validation_error=False)
-
+        flaw = FlawFactory(major_incident_state=Flaw.FlawMajorIncident.APPROVED)
         AffectFactory(flaw=flaw)
-        FlawMetaFactory(
-            flaw=flaw,
-            type=FlawMeta.FlawMetaType.REQUIRES_SUMMARY,
-            meta_attr={"status": "-"},
-        )
-
-        assert flaw.classification == {
-            "workflow": "major incident workflow",
-            "state": "DRAFT",
-        }
 
-        flaw.is_major_incident = Flaw.FlawMajorIncident.NOVALUE
+        # TODO remove this 3 lines when major incident property is recreated
+        flaw.is_major_incident = True
+        flaw.save()
```

Comment:
why? IIRC `flaw.is_major_incident` is no longer used and `major_incident_state` is the canonical source of data

Reply:
OSIM currently only accepts properties and by the time I developed it, I didn't wanted to duplicate this function with an attribute since it has a comment saying it will be converted into attr in future... I believe the temporary solution for this would extend OSIM to accept the function `is_major_incident_temp` as a value. Please take a look in the new changes I made for this commit in `apps/osim/checks.py::CheckParser::desc2property` and see if it would be an acceptable solution.